src/screens/DashboardScreen/DashboardScreen.tsx

```diff
@@ -4,14 +4,20 @@ import { View, Text } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
 import useGlobal from '@state';
 import { Title, SpacerInline, Header } from '@elements';
-import DonationsOrClaims from './DonationsOrClaims';
+import DonationsOrClaims from '../../library/DonationsOrClaims';
 import styles from './DashboardScreen.styles';
+import DonorDashboardScreen from '../DonorDashboardScreen';
 
 const DashboardScreen = () => {
 	const { navigate } = useNavigation();
 	const [ state ] = useGlobal();
 	const { userIdentity } = state;
-	const title = userIdentity === 'donor' ? 'My Donations.' : 'Open Donations.';
+
```

Comment:
I like this. I wonder, for the future, if this Screen should just act as a conditional in pointing to either `DonorDashboardScreen` or a potential `ClientDashboardScreen`

Reply:
Yes, a near-future plan should be to create a well-defined `ClientDashboardScreen`.